### PR TITLE
Added the "placeholder" tag

### PR DESCRIPTION
**Pull request by Andrew-Morozko**
Currently, the data placeholder in help is named as uppercased `spec.long`. This looks bad and redundant if `spec.long` is a long name. Here's an example from my project:
```
Options:
  --parallel-uploads PARALLEL-UPLOADS, -j PARALLEL-UPLOADS
                         maximum number of concurrent uploads
```

I propose adding optional `dataname:NAME` tag that overrides this behavior, so you can do this:
```
Options:
  --parallel-uploads N, -j N   maximum number of concurrent uploads
```

This change will also allow to better explain the command (sometimes).

Before:
```
   --optimize OPTIMIZE, -O OPTIMIZE
                          optimization level
```
After:
```
   --optimize LEVEL, -O LEVEL
                          <the help string isn't needed anymore>
```

I'm having doubts about the name of the `dataname` tag, but that's the best name I could come up with;)

### CODE REVIEW

File: example_test.go [hunk 1]
```diff
@@ -135,6 +135,42 @@ func Example_helpText() {
 	//   --help, -h             display this help and exit
 }
 
+// This example shows the usage string generated by go-arg with custom datanames
+func Example_helpDataname() {
+	// These are the args you would pass in on the command line
+	os.Args = split("./example --help")
+
+	var args struct {
```
Comment (alexflint) [inline example_test.go]: I'd suggest removing all fields that don't use the new tag from this example. Also, would you add a slide-valued field that uses the new tag?

**Comment by Andrew-Morozko** [comment]
@alexflint Implemented your changes, `placeholder` is totally a better name. Also added the parser test for `placeholder` tag and a section to the readme, IDK if it is needed.

File: README.md [hunk 2]
```diff
@@ -148,6 +148,32 @@ var args struct {
 arg.MustParse(&args)
 ```
 
+### Custom placeholders
+
```
Comment (alexflint) [inline README.md]: Add a little text here: "Use the placeholder tag to control which placeholder text is used in the usage text"

File: README.md [hunk 3]
```diff
@@ -148,6 +148,32 @@ var args struct {
 arg.MustParse(&args)
 ```
 
+### Custom placeholders
```
Comment (alexflint) [inline README.md]: Can you move this section down below custom parsing?

File: example_test.go [hunk 4]
```diff
@@ -135,6 +135,38 @@ func Example_helpText() {
 	//   --help, -h             display this help and exit
 }
 
+// This example shows the usage string generated by go-arg with customized placeholders
+func Example_helpDataname() {
```
Comment (alexflint) [inline example_test.go]: `func Example_helpPlaceholder`

**Comment by Andrew-Morozko** [comment]
Alright, I think this PR is ready. Thank you for the cool library and for the code reviews @alexflint

**Comment by alexflint** [comment]
Thank you @Andrew-Morozko

**Pull request merged**